modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -393,7 +393,11 @@ def buildProject(sassLint = true) {
         [$class: 'StringParameterDefinition',
           name: 'SCHEMA_BRANCH',
```

Comment:
I wonder if we could rid of this parameter and use `SCHEMA_COMMIT` everywhere instead?